_includes/initial-login-changepw.md

```diff
@@ -1,8 +1,7 @@
 Change your password to ensure more private and secure access to
 {{ site.data.product.title_short }}.
 
-1.  Browse to the URL for the login screen. (<https://xx.xx.xx.xx> on
-    the virtual machine instance)
+1.  Browse to the URL for the login screen.
```

Comment:
@Fryguy how can we improve this line?

Reply:
The parenthesis shows an example, so maybe adding an `e.g.`? I'm not sure what was wrong with it originally.

Reply:
Yeah I think the original was okay, maybe something to say "get the ipaddress or hostname from the VM details page"

Reply:
I meant like in terms of navigation, how does a user get the URL?

Reply:
It is going to be different on every platform (vmware, amazon, google, etc...) but all of them will show you the IP address and/or dns name of the VM in some way.

Reply:
Hmm for example this doc is for GCP, I am still not sure how exactly user will get the URL for the login screen.

Reply:
This doc isn't for GCP, it is included into all of the "installing_on_*" docs,
```
$ grep -r initial-login-changepw
installing_on_scvmm/index.md:{% include initial-login-changepw.md %}
installing_on_vmware_vsphere/index.md:{% include initial-login-changepw.md %}
installing_on_microsoft_azure/index.md:{% include initial-login-changepw.md %}
installing_on_google_compute_engine/index.md:{% include initial-login-changepw.md %}
installing_on_red_hat_virtualization/index.md:{% include initial-login-changepw.md %}
grep: .git/index: binary file matches
installing_on_red_hat_enterprise_linux_openstack_platform/index.md:{% include initial-login-changepw.md %}
```
Which was the point of my comment :point_down: https://github.com/ManageIQ/manageiq-documentation/pull/1805#issuecomment-2289091895

Reply:
IMO I think it is reasonable to assume that a user will know how to get the IP address of the VM that they just deployed on whatever platform they're using.

Reply:
ok thanks Adam.